app/models/claim/advocate_claim.rb

```diff
@@ -72,7 +72,7 @@ class AdvocateClaim < BaseClaim
              dependent: :destroy,
              inverse_of: :claim,
              validate: proc { |claim| claim.step_validation_required?(:basic_fees) }
-    has_many :fixed_fees, -> { order(:created_at) },
```

Comment:
**not a blocker** but @colinbruce this is the other one i meant to talk to you about. There is a `position` attribute on `fee_types` that could be used to control this in a more controlled/description way.

Reply:
Agreed.  I think this change is a hangover of an attempt to sequence in-memory objects.  As long as the `position` field doesn't affect the rendering on the page it should be fine.